src/core/Jalasoft.TeamUp.Projects.Core/Validators/InvitationValidator.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Jalasoft.TeamUp.Projects.Core.Validators
+{
+    using System;
+    using FluentValidation;
+    using Jalasoft.TeamUp.Projects.Models;
+
+    public class InvitationValidator : AbstractValidator<Invitation>
+    {
+        public InvitationValidator()
+        {
+            this.RuleFor(invitation => invitation.Id)
+                .NotEmpty().NotNull();
```

Comment:
```suggestion
                .NotEmpty();
```
is enough with .NotEmpty() only
https://docs.fluentvalidation.net/en/latest/built-in-validators.html#notempty-validator

Reply:
done